.circleci/config.yml

```diff
@@ -50,6 +50,7 @@ commands:
 jobs:
   build-and-test:
     <<: *node_executor
+    resource_class: "medium+"
```

Comment:
Needed because it seems like CircleCI runs out of RAM otherwise, similar to https://discuss.circleci.com/t/circleci-builds-and-runs-some-tests-successfully-but-then-freezes/31243